test.js

```diff
@@ -29,6 +29,48 @@ tape('api: store.model()', (t) => {
 })
 
 tape('api: store.use()', (t) => {
+  t.test('should allow model injection', (t) => {
+    t.plan(1)
+    const store = barracks()
+    store.model({
+      state: {
+        accessed: false
+      }
+    })
+    store.use({
+      injectModels: [
```

Comment:
I think the "inject" word sounds scary; I never use it in day to day life. Maybs we could name it "addModels" instead?